package-overrides/github/kenwheeler/slick@1.5.8.json

```diff
@@ -0,0 +1,18 @@
+{
+  "main": "slick",
+  "files": null,
+  "directories": {
+    "lib": "slick"
+  },
+  "shim": {
+    "slick": {
+      "deps": [
+        "jquery"
+      ],
+      "exports": "$"
+    }
+  },
+  "dependencies": {
+    "jquery": "github:components/jquery"
```

Comment:
Perhaps add a `registry: 'jspm'` and reference `jquery: '*'`. If we ever move jQuery, it could be useful to be referencing the jspm registry this way.